src/PubSubClient.cpp

```diff
@@ -588,3 +602,10 @@ PubSubClient& PubSubClient::setStream(Stream& stream){
 int PubSubClient::state() {
     return this->_state;
 }
+
+PubSubClient& PubSubClient::setPacketBuffer(uint8_t * pBuf, uint16_t length){
+  MQTT_MAX_PACKET_SIZE = length;
```

Comment:
Looks like assignment to a global/define due to the uppercasing. The instance variable should probably follow the regular conventions (camelCase). For instance `bufferLength`